src/PythonQtClassInfo.cpp

```diff
@@ -49,6 +49,11 @@
 
 QHash<QByteArray, int> PythonQtMethodInfo::_parameterTypeDict;
 
+// List of words that are reserved in Python, but not in C++, so they need escaping
+QSet<QByteArray> PythonQtClassInfo::_reservedNames{
+  "None", "True", "False"
+};
```

Comment:
I guess this is not a full list, just the names that are reasonably can be used in specifiers - otherwise there would be a few missing (like "elif", "with" or "pass").

Reply:
Exactly. Only the names that would reasonably occur in an enum value name (because e.g. they start with a upper case letter).